ios/FluentUI/Button/ButtonTokenSet.swift

```diff
@@ -0,0 +1,262 @@
+                        // textDisabled
+                        return DynamicColor(light: ColorValue(0xACACAC) /* gray300 */,
+                                            lightHighContrast: ColorValue(0x6E6E6E) /* gray500 */,
+                                            dark: ColorValue(0x404040) /* gray600 */,
+                                            darkHighContrast: ColorValue(0x919191) /* gray400 */)
+                    }
+                }
+            case .foregroundPressedColor:
+                return .dynamicColor {
+                    switch style() {
+                    case .primaryFilled, .dangerFilled:
+                        return theme.aliasTokens.foregroundColors[.neutralInverted]
+                    case .primaryOutline, .secondaryOutline, .tertiaryOutline, .borderless:
+                        return theme.aliasTokens.brandColors[.tint20]
+                    case .dangerOutline:
+                        // dangerTint20
+                        return DynamicColor(light: ColorValue(0xE87979),
+                                            dark: ColorValue(0x8B2323))
+                    }
+                }
+            case .titleFont:
+                return .fontInfo {
+                    switch style() {
+                    case .primaryFilled, .primaryOutline, .dangerFilled, .dangerOutline, .borderless:
+                        return theme.aliasTokens.typography[.body2]
+                    case .secondaryOutline, .tertiaryOutline:
+                        return theme.aliasTokens.typography[.caption1]
+                    }
+                }
+            }
+        }
+    }
+
+    var style: () -> ButtonStyle
+}
+
+extension ButtonTokenSet {
+    /// The value for the horizontal padding between the content of the button and the frame.
+    static func horizontalPadding(_ style: ButtonStyle) -> CGFloat {
+        switch style {
+        case .dangerFilled, .dangerOutline, .primaryFilled, .primaryOutline:
+            return GlobalTokens.spacing(.large)
+        case .secondaryOutline:
+            return 14.0
+        case .borderless:
+            return GlobalTokens.spacing(.small)
+        case .tertiaryOutline:
+            return GlobalTokens.spacing(.xSmall)
+        }
+    }
+
+    /// The minimum value for the height of the content of the button.
+    static func minContainerHeight(_ style: ButtonStyle) -> CGFloat {
+        switch style {
+        case .borderless, .dangerFilled, .dangerOutline, .primaryFilled, .primaryOutline:
+            return 20
+        case .secondaryOutline, .tertiaryOutline:
+            return 18
+        }
+    }
+
+    /// The value for the spacing between the title and image.
+    static func titleImageSpacing(_ style: ButtonStyle) -> CGFloat {
+        switch style {
+        case .dangerFilled, .dangerOutline, .primaryFilled, .primaryOutline:
+            return 10
```

Comment:
and we are using 10 instead of globaltoken.spacing because we don't have a token for 10 yet?

Reply:
Yeah, xSmall is 8 and small is 12.